script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -727,56 +853,68 @@ The monastery.
 [sha:bd1bc148859795cc327d0e10ad2d86829c6c5e18]{
 -- Page 477, Offset 19462.
 -- それは教会の教えのもと、俗世と離れて共同生活を営むコトを選んだ信徒たちの施設である。
-An institution for those among the faithful who have chosen to lead a communal life, removed from the earthly world and under the teaching of the Church.
+An institution for those among the faithful who wish to withdraw from the earthly world,
+ and live communally under the teachings of the Church.
 }
 [sha:d0181ef4ae26873c138c9a9fd33826704e3c51fe]{
 -- Page 477, Offset 19463.
 -- 法王庁が世界でもっとも小さな国であるように、修道院もそれぞれが独立した閉鎖世界といっていい。
-Just as the Papacy constitutes the smallest country on Earth, it could be said that each monastery represents an independent, detached world of its own.
+// Assuming it is a singular Monastery
+Just as the Vatican constitutes the smallest country on earth,
+ it could be said that the monastery represents an independent world in its own right,
+ sealed off from the rest of society.
 }
 [sha:5dcfcaf28685addb3815503e5075d151b796f15d]{
 -- Page 478, Offset 19464.
 -- 修道女はその後の一生を院内で完結させる事を誓約し、この完成された共同体の一員となる。
-The nuns pledge to spend the rest of their lives inside the monastery, becoming members of this perfected collective.
+Nuns who pledge to spend the remainder of their lives inside its walls become members of this perfected collective.
 }
 [sha:7ccc13bf103c9085c4c29fbf8acb87ae28499ad3]{
 -- Page 478, Offset 19465.
 -- 彼らは外の世界と壁一枚を隔て、清貧・貞潔・従順を守りながら、父の子として相応しい一生を終える。
-Walled out from the outside world, they lead a life of poverty, purity and obedience and pass away in a manner befitting for Children of the Father.
+Cut off from the outside world,
+ they live out their days upholding the ideals of purity, obedience,
+ and freedom from material desires, as befits children of the Heavenly Father.
 }
 [sha:ffb42e4921d1fdb8cc859773eb8ea41259d4ac15]{
 -- Page 479, Offset 19466.
 -- 望んで修道院の門を叩く者はいい。
-Suitable for those who willingly knock on the doors of the monastery.
+For those who knock willingly on the doors of the monastery, it is a worthy life.
 }
 [sha:92df72fd036e6d2255dc76930bc32ad60a0bc9af]{
 -- Page 479, Offset 19467.
 -- しかしこの閉ざされた世界を望まない者にとって、修道院送りは拷問に近い。
-But for those who do not wish to be sealed in these closed worlds, being sent to a monastery is akin to torture.
+But for those who do not wish to be sealed away in this cloistered world,
+ being sent to the monastery is akin to torture.
 }
 [sha:221bc8d865f9dfefc93d5bdd4080a02185b8c0fd]{
 -- Page 480, Offset 19468.
 -- ある理由から<彼女|ノエル>には代行者以外の道が閉ざされていた。
-Due to various reasons, no other path than that of the Executor is open to Noel.
+Due to her circumstances, the only path not denied to Noel is that of an Executor.
```

Comment:
maybe make the ある理由 stick out a bit more in English

> Due to her peculiar circumstances...

Because it's meant to come across as a bit of a foreshadowing.